src/main/java/frc/robot/subsystems/BatterySubsystem.java

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+// import edu.wpi.first.wpilibj2.*;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.RobotController;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+/** Add your docs here. */
+// Put methods for controlling this subsystem
+// here. Call these from Commands.
+public class BatterySubsystem extends SubsystemBase {
+
+  private ShuffleboardTab btTab;
+  private NetworkTableEntry sbVoltage;
+  private NetworkTableEntry sbInCurrent;
+  private NetworkTableEntry sbTimer;
+  private NetworkTableEntry sbTimerChange;
+  private Timer timer;
+
+  private void init() {
+    timer.reset();
```

Comment:
timer = new Timer();

This code only counts the time since the robot code reset the timer.  Can we get since the robot was turned on?

Reply:
Completed, moved the reset to robotInit()